### ISSUE TITLE
Nami freezing up after typing over a certain number

### ISSUE THREAD

**Issue by jmoyns** [berry-pool/nami#780]
I am trying to liquidate some ada by sending to an exchange. I am trying to send 190000ADA to my coinbase wallet. After I type beyond 190 it freezes up indefinitely. Not sure what to do. Thanks!
<img width="370" alt="Screen Shot 2023-01-08 at 12 06 45 PM" src="https://user-images.githubusercontent.com/122232141/211216613-8452f191-ca5a-4c4b-b28a-962e07a27a70.png">

**Comment by mark-EthDev** [comment]
@jmoyns Talk to Cardano Support on the [Live support page](http://explore-cardano.org/) so that your issue can be looked into.

**Comment by jmoyns** [comment]
I'm not familiar with Cardano Support. They can help with Nami issues?

**Comment by mark-EthDev** [comment]
Yes you should create chat with the live support and explain your issues to them.

**Comment by jmoyns** [comment]
You sent me to a website that spells Cardano with two n's

**Comment by GGAlanSmithee** [comment]
@jmoyns as you've already identified, these are scammers
